cli_tools/common/utils/logging/service/logger.go

```diff
@@ -229,7 +232,9 @@ func (l *Logger) runWithServerLogging(function func() (Loggable, error),
 		l.logStart()
 	}()
 
-	loggable, err := function()
+	capture := &safeRunner{inner: function}
```

Comment:
why is safeRunner variable called capture? Seems unintuitive.

Reply:
Changed to `r` ("capture" was intended to signify the captured loggable and error)